ilc/server/tailor/configs-injector.spec.js

```diff
@@ -201,7 +203,7 @@ describe('configs injector', () => {
         };
 
         const getSpaConfig = () => {
-            const pickApp = _.pick(['spaBundle', 'cssBundle', 'dependencies', 'props', 'kind']);
+            const pickApp = _fp.pick(['spaBundle', 'cssBundle', 'dependencies', 'props', 'kind', 'wrappedWith']);
```

Comment:
do we really need here `lodash/fp`?

Reply:
It's convenient here... Easier to keep, IMO